app/models/user.rb

```diff
@@ -5,7 +5,6 @@ class User < ActiveRecord::Base
   MAX_USERNAME_LEN = 80
 
   attr_accessor :password, :password_confirmation
-  attr_protected :crypted
```

Comment:
We're using strong params for `User` now so `:crypted` is just not on the whitelist